src/app/config/thunks.js

```diff
@@ -120,3 +118,145 @@ export const createCollectionRequest = collection => (dispatch, getState) => {
             console.error(error);
         });
 };
+
+export const getUsersRequest = () => dispatch => {
+    users
+        .getAll({ active: true })
+        .then(response => {
+            dispatch(getUsersRequestSuccess(response));
+        })
+        .catch(error => {
+            // TODO move, handling errors should be done elsewhere see note in createCollectionRequest func above
+            if (error.status != null) {
+                if (error.status >= 400 && error.status < 500) {
+                    switch (error.status) {
+                        case 400: {
+                            const notification = {
+                                type: "warning",
+                                message: errCodes.GET_USERS_UNEXPECTED_FILTER_ERROR,
+                                autoDismiss: 5000,
+                            };
+                            notifications.add(notification);
+                            break;
+                        }
+                        case 404: {
+                            const notification = {
+                                type: "warning",
+                                message: errCodes.GET_USERS_NOT_FOUND,
+                                autoDismiss: 5000,
+                            };
+                            notifications.add(notification);
+                            break;
+                        }
+                        default: {
+                            const notification = {
+                                type: "warning",
+                                message: errorCodes.GET_USERS_UNEXPECTED_ERROR_SHORT,
+                                isDismissable: true,
+                            };
+                            notifications.add(notification);
+                            break;
+                        }
+                    }
+                } else {
+                    const notification = {
+                        type: "warning",
+                        message: errCodes.GET_USERS_UNEXPECTED_ERROR_SHORT,
+                        isDismissable: true,
+                    };
+                    notifications.add(notification);
+                }
+            } else {
+                const notification = {
+                    type: "warning",
+                    message: errCodes.GET_USERS_UNEXPECTED_ERROR_SHORT,
+                    isDismissable: true,
+                };
+                notifications.add(notification);
+            }
+            console.error(error);
```

Comment:
shouldn't log to the console in completed code

Reply:
I agree with Dan, no harm in having this here